images/update-drivers-website/.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
Good catch!

Reply:
Mmh do we need this gitignore? Nothing is being pushed to test-infra :/